config/initializers/doorkeeper.rb

```diff
@@ -0,0 +1,106 @@
+Doorkeeper.configure do
+  # Change the ORM that doorkeeper will use.
+  # Currently supported options are :active_record, :mongoid2, :mongoid3,
+  # :mongoid4, :mongo_mapper
+  orm :active_record
+
+  # This block will be called to check whether the resource owner is authenticated or not.
+  resource_owner_authenticator do
+    # fail "Please configure doorkeeper resource_owner_authenticator block located in #{__FILE__}"
+    # Put your resource owner authentication logic here.
+    # Example implementation:
+    # User.find_by_id(session[:user_id]) || redirect_to(new_user_session_url)
+  end
+
+  # authenticate based on username / password credentials
+  resource_owner_from_credentials do |routes|
+    User.try_to_login(params[:username], params[:password])
+  end
```

Comment:
I added the blocked `resource_owner_from_credentials` which calls `User.try_to_login`. I used params[:username] rather than params[:login] to be consistent with what is sent by [ember-simple-auth-oauth2](https://github.com/simplabs/ember-simple-auth/tree/master/packages/ember-simple-auth-oauth2)